README.md

```diff
@@ -176,6 +176,20 @@ Please take a look at [Our costumers](https://github.com/skywinder/web3swift/wik
 - If you'd like to **see web3swift best practices**, check [Projects that using web3swift](https://github.com/skywinder/web3swift/wiki/Our-Customers).
 - If you **found a bug**, [open an issue](https://github.com/skywinder/web3swift/issues).
 
+## Development
+To do local development and run the local tests, we recommend to use [ganache](https://github.com/trufflesuite/ganache) which is also used by CI when running github actions.
+
+```cli
+// To install
+$ npm install ganache --global
+
+// To run
+$ ganache
+
```

Comment:
Drop this last empty line please as right now it just takes more space in the README.

Reply:
![Screenshot 2022-11-10 at 15 38 58](https://user-images.githubusercontent.com/36865532/201106476-cd677134-bb15-4a0b-aff3-465217441548.png)